t/class/accessor.t

```diff
@@ -0,0 +1,46 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    require './test.pl';
+    set_up_inc('../lib');
+    require Config;
+}
+
+use v5.36;
+use feature 'class';
+no warnings 'experimental::class';
+
+# reader accessors
+{
+    class Test1 {
+        field $s :reader = "the scalar";
+        field @a :reader = qw( the array );
+        field %h :reader = qw( the hash );
+    }
+
+    my $o = Test1->new;
+    is($o->s, "the scalar", '$o->s accessor');
+    ok(eq_array([$o->a], [qw( the array )]), '$o->a accessor');
+    ok(eq_hash({$o->h}, {qw( the hash )}), '$o->h accessor');
+
+    is(scalar $o->a, 2, '$o->a accessor in scalar context');
+    is(scalar $o->h, 1, '$o->h accessor in scalar context');
+
+    # Read accessor does not permit arguments
+    ok(!eval { $o->s("value") },
+        'Reader accessor fails with argument');
+    like($@, qr/^Too many arguments for subroutine \'Test1::s\' \(got 1; expected 0\) at /,
+        'Failure from argument to accessor');
+}
+
+# Alternative names
+{
+    class Test2 {
+        field $f :reader(get_f) = "value";
+    }
+
+    is(Test2->new->get_f, "value", 'accessor with altered name');
```

Comment:
Should `Test2->new->f` also work here? The POD suggests not but doesn't state it explicitly and there's no test confirming it doesn't.

Reply:
Test added